diff -burNp zebra-0.95a/bgpd/bgpd.h zebra-0.95a-dud/bgpd/bgpd.h
--- zebra-0.95a/bgpd/bgpd.h	2005-08-05 12:18:33.000000000 +0200
+++ zebra-0.95a-dud/bgpd/bgpd.h	2009-01-23 09:32:41.733189182 +0100
@@ -700,7 +700,7 @@ struct bgp_nlri
 
 /* Default configuration settings for bgpd.  */
 #define BGP_VTY_PORT                          2605
-#define BGP_VTYSH_PATH                "/tmp/.bgpd"
+#define BGP_VTYSH_PATH                "/var/run/zebra/bgpd"
 #define BGP_DEFAULT_CONFIG             "bgpd.conf"
 
 /* Check AS path loop when we send NLRI.  */
diff -burNp zebra-0.95a/ospfd/ospfd.h zebra-0.95a-dud/ospfd/ospfd.h
--- zebra-0.95a/ospfd/ospfd.h	2003-03-28 20:55:29.000000000 +0100
+++ zebra-0.95a-dud/ospfd/ospfd.h	2009-01-23 09:34:50.019847042 +0100
@@ -39,7 +39,7 @@
 
 /* VTY port number. */
 #define OSPF_VTY_PORT          2604
-#define OSPF_VTYSH_PATH        "/tmp/.ospfd"
+#define OSPF_VTYSH_PATH        "/var/run/zebra/ospfd"
 
 /* IP TTL for OSPF protocol. */
 #define OSPF_IP_TTL             1
diff -burNp zebra-0.95a/ripd/ripd.h zebra-0.95a-dud/ripd/ripd.h
--- zebra-0.95a/ripd/ripd.h	2004-06-01 07:08:47.000000000 +0200
+++ zebra-0.95a-dud/ripd/ripd.h	2009-01-23 09:35:57.797127225 +0100
@@ -68,7 +68,7 @@
 /* RIP port number. */
 #define RIP_PORT_DEFAULT               520
 #define RIP_VTY_PORT                  2602
-#define RIP_VTYSH_PATH        "/tmp/.ripd"
+#define RIP_VTYSH_PATH        "/var/run/zebra/ripd"
 
 /* Default configuration file name. */
 #define RIPD_DEFAULT_CONFIG    "ripd.conf"
diff -burNp zebra-0.95a/ripngd/ripngd.h zebra-0.95a-dud/ripngd/ripngd.h
--- zebra-0.95a/ripngd/ripngd.h	2001-12-19 21:31:38.000000000 +0100
+++ zebra-0.95a-dud/ripngd/ripngd.h	2009-01-23 09:40:52.567514443 +0100
@@ -27,7 +27,7 @@
 #define RIPNG_V1                         1
 #define RIPNG_PORT_DEFAULT             521
 #define RIPNG_VTY_PORT                2603
-#define RIPNG_VTYSH_PATH              "/tmp/.ripngd"
+#define RIPNG_VTYSH_PATH              "/var/run/zebra/ripngd"
 #define RIPNG_MAX_PACKET_SIZE         1500
 #define RIPNG_PRIORITY_DEFAULT           0
 
diff -burNp zebra-0.95a/vtysh/vtysh.h zebra-0.95a-dud/vtysh/vtysh.h
--- zebra-0.95a/vtysh/vtysh.h	2003-11-04 20:10:18.000000000 +0100
+++ zebra-0.95a-dud/vtysh/vtysh.h	2009-01-23 09:47:17.419767559 +0100
@@ -40,12 +40,12 @@
 #define VTYSH_INDEX_MAX   6
 
 /* UNIX domain socket path. */
-#define ZEBRA_PATH "/tmp/.zebra"
-#define RIP_PATH "/tmp/.ripd"
-#define RIPNG_PATH "/tmp/.ripngd"
-#define OSPF_PATH "/tmp/.ospfd"
-#define OSPF6_PATH "/tmp/.ospf6d"
-#define BGP_PATH "/tmp/.bgpd"
+#define ZEBRA_PATH "/var/run/zebra/zebra"
+#define RIP_PATH "/var/run/zebra/ripd"
+#define RIPNG_PATH "/var/run/zebra/ripngd"
+#define OSPF_PATH "/var/run/zebra/ospfd"
+#define OSPF6_PATH "/var/run/zebra/ospf6d"
+#define BGP_PATH "/var/run/zebra/bgpd"
 
 /* vtysh local configuration file. */
 #define VTYSH_DEFAULT_CONFIG "vtysh.conf"
diff -burNp zebra-0.95a/zebra/zserv.h zebra-0.95a-dud/zebra/zserv.h
--- zebra-0.95a/zebra/zserv.h	2001-01-16 00:25:24.000000000 +0100
+++ zebra-0.95a-dud/zebra/zserv.h	2009-01-23 09:43:46.633330294 +0100
@@ -25,8 +25,8 @@
 /* Default port information. */
 #define ZEBRA_PORT                    2600
 #define ZEBRA_VTY_PORT                2601
-#define ZEBRA_VTYSH_PATH              "/tmp/.zebra"
-#define ZEBRA_SERV_PATH               "/tmp/.zserv"
+#define ZEBRA_VTYSH_PATH              "/var/run/zebra/zebra"
+#define ZEBRA_SERV_PATH               "/var/run/zebra/zserv"
 
 /* Default configuration filename. */
 #define DEFAULT_CONFIG_FILE "zebra.conf"
