diff -urN zebra-0.73.orig/configure.in zebra-0.73/configure.in
--- zebra-0.73.orig/configure.in	Wed Jul 28 18:56:17 1999
+++ zebra-0.73/configure.in	Wed Jul 28 19:00:00 1999
@@ -320,9 +320,13 @@
 dnl ----------
 dnl Linux IPv6
 dnl ----------
-if test x`ls /proc/net/ipv6_route 2>/dev/null` = x"/proc/net/ipv6_route"
-then
-   zebra_cv_ipv6=yes
+AC_EGREP_CPP(yes, [dnl
+#include <linux/version.h>
+/* 2.1.128 or later */
+#if LINUX_VERSION_CODE >= 0x020180
+yes
+#endif],
+   [zebra_cv_ipv6=yes
    AC_DEFINE(HAVE_IPV6)
    AC_DEFINE(LINUX_IPV6)   
    RIPNGD="ripngd"
@@ -331,8 +335,7 @@
    if test x`ls /usr/inet6/lib/libinet6.a 2>/dev/null` != x;then
      LIB_IPV6="-L/usr/inet6/lib -linet6"
    fi
-   AC_MSG_RESULT(Linux IPv6)
-fi
+   AC_MSG_RESULT(Linux IPv6)])
 
 dnl -----------------------
 dnl Set IPv6 related values
