diff -Naur zebra-0.92a/configure.in zebra-0.92a-p/configure.in
--- zebra-0.92a/configure.in	Thu Nov 29 14:18:40 2001
+++ zebra-0.92a-p/configure.in	Thu Nov 29 14:19:24 2001
@@ -106,7 +106,7 @@
   AC_DEFINE(HAVE_NSSA)
 fi
 
-changequote(, )dnl
+changequote(, )
 
 MULTIPATH_NUM=1
 
@@ -122,7 +122,7 @@
     ;;
 esac
 
-changequote([, ])dnl
+changequote([, ])
 
 AC_SUBST(MULTIPATH_NUM)
 
@@ -205,7 +205,7 @@
 if test "$with_libpam" = "yes"; then
 dnl took this test from proftpd's configure.in and suited to our needs
 dnl -------------------------------------------------------------------------
-dnl
+
 dnl This next check looks funky due to a linker problem with some versions
 dnl of the PAM library.  Prior to 0.72 release, the Linux PAM shared library
 dnl omitted requiring libdl linking information. PAM-0.72 or better ships
@@ -420,7 +420,7 @@
 dnl Linux IPv6
 dnl ----------
 if test "${enable_ipv6}" = "yes"; then
-   AC_EGREP_CPP(yes, [dnl
+   AC_EGREP_CPP(yes, [
    #include <linux/version.h>
    /* 2.1.128 or later */
    #if LINUX_VERSION_CODE >= 0x020180
@@ -720,10 +720,7 @@
 dnl set paths for process id files
 dnl ------------------------------
 AC_CACHE_CHECK(pid file directory,ac_piddir,
-[for ZEBRA_PID_DIR in /var/run	dnl
-                   /var/adm	dnl
-                   /etc		dnl
-                   /dev/null;
+[for ZEBRA_PID_DIR in /var/run /var/adm /etc /dev/null;
 do
   test -d $ZEBRA_PID_DIR && break
 done
