diff -durN zebra-0.93.orig/bgpd/bgpd.h zebra-0.93/bgpd/bgpd.h
--- zebra-0.93.orig/bgpd/bgpd.h	Sat Jun 29 08:20:39 2002
+++ zebra-0.93/bgpd/bgpd.h	Sun Jul  7 17:56:01 2002
@@ -608,7 +608,7 @@
 
 /* Default configuration settings for bgpd.  */
 #define BGP_VTY_PORT                          2605
-#define BGP_VTYSH_PATH                "/tmp/.bgpd"
+#define BGP_VTYSH_PATH                "/var/run/zebra/bgpd"
 #define BGP_DEFAULT_CONFIG             "bgpd.conf"
 
 /* Check AS path loop when we send NLRI.  */
diff -durN zebra-0.93.orig/ospf6d/ospf6d.h zebra-0.93/ospf6d/ospf6d.h
--- zebra-0.93.orig/ospf6d/ospf6d.h	Sun Jul  7 10:12:48 2002
+++ zebra-0.93/ospf6d/ospf6d.h	Sun Jul  7 17:54:52 2002
@@ -96,7 +96,7 @@
 
 /* Default port values. */
 #define OSPF6_VTY_PORT             2606
-#define OSPF6_VTYSH_PATH           "/tmp/.ospf6d"
+#define OSPF6_VTYSH_PATH           "/var/run/zebra/ospf6d"
 
 #ifdef INRIA_IPV6
 #ifndef IPV6_PKTINFO
diff -durN zebra-0.93.orig/ospfd/ospfd.h zebra-0.93/ospfd/ospfd.h
--- zebra-0.93.orig/ospfd/ospfd.h	Sat Jul  6 09:36:59 2002
+++ zebra-0.93/ospfd/ospfd.h	Sun Jul  7 17:54:52 2002
@@ -34,7 +34,7 @@
 
 /* VTY port number. */
 #define OSPF_VTY_PORT          2604
-#define OSPF_VTYSH_PATH        "/tmp/.ospfd"
+#define OSPF_VTYSH_PATH        "/var/run/zebra/ospfd"
 
 /* IP TTL for OSPF protocol. */
 #define OSPF_IP_TTL             1
diff -durN zebra-0.93.orig/ripd/ripd.h zebra-0.93/ripd/ripd.h
--- zebra-0.93.orig/ripd/ripd.h	Sat Jan 27 10:24:23 2001
+++ zebra-0.93/ripd/ripd.h	Sun Jul  7 17:54:52 2002
@@ -64,7 +64,7 @@
 /* RIP port number. */
 #define RIP_PORT_DEFAULT               520
 #define RIP_VTY_PORT                  2602
-#define RIP_VTYSH_PATH        "/tmp/.ripd"
+#define RIP_VTYSH_PATH        "/var/run/zebra/ripd"
 
 /* Default configuration file name. */
 #define RIPD_DEFAULT_CONFIG    "ripd.conf"
diff -durN zebra-0.93.orig/ripngd/ripngd.h zebra-0.93/ripngd/ripngd.h
--- zebra-0.93.orig/ripngd/ripngd.h	Wed Dec 19 21:31:38 2001
+++ zebra-0.93/ripngd/ripngd.h	Sun Jul  7 17:54:52 2002
@@ -27,7 +27,7 @@
 #define RIPNG_V1                         1
 #define RIPNG_PORT_DEFAULT             521
 #define RIPNG_VTY_PORT                2603
-#define RIPNG_VTYSH_PATH              "/tmp/.ripngd"
+#define RIPNG_VTYSH_PATH              "/var/run/zebra/ripngd"
 #define RIPNG_MAX_PACKET_SIZE         1500
 #define RIPNG_PRIORITY_DEFAULT           0
 
diff -durN zebra-0.93.orig/vtysh/vtysh.h zebra-0.93/vtysh/vtysh.h
--- zebra-0.93.orig/vtysh/vtysh.h	Sun Feb 11 04:37:34 2001
+++ zebra-0.93/vtysh/vtysh.h	Sun Jul  7 17:54:52 2002
@@ -40,12 +40,12 @@
 #define VTYSH_INDEX_MAX   6
 
 /* UNIX domain socket path. */
-#define ZEBRA_PATH "/tmp/.zebra"
-#define RIP_PATH "/tmp/.ripd"
-#define RIPNG_PATH "/tmp/.ripngd"
-#define OSPF_PATH "/tmp/.ospfd"
-#define OSPF6_PATH "/tmp/.ospf6d"
-#define BGP_PATH "/tmp/.bgpd"
+#define ZEBRA_PATH "/var/run/zebra/zebra"
+#define RIP_PATH "/var/run/zebra/ripd"
+#define RIPNG_PATH "/var/run/zebra/ripngd"
+#define OSPF_PATH "/var/run/zebra/ospfd"
+#define OSPF6_PATH "/var/run/zebra/ospf6d"
+#define BGP_PATH "/var/run/zebra/bgpd"
 
 /* vtysh local configuration file. */
 #define VTYSH_DEFAULT_CONFIG "vtysh.conf"
diff -durN zebra-0.93.orig/zebra/zserv.h zebra-0.93/zebra/zserv.h
--- zebra-0.93.orig/zebra/zserv.h	Tue Jan 16 00:25:24 2001
+++ zebra-0.93/zebra/zserv.h	Sun Jul  7 17:54:52 2002
@@ -25,8 +25,8 @@
 /* Default port information. */
 #define ZEBRA_PORT                    2600
 #define ZEBRA_VTY_PORT                2601
-#define ZEBRA_VTYSH_PATH              "/tmp/.zebra"
-#define ZEBRA_SERV_PATH               "/tmp/.zserv"
+#define ZEBRA_VTYSH_PATH              "/var/run/zebra/zebra"
+#define ZEBRA_SERV_PATH               "/var/run/zebra/zserv"
 
 /* Default configuration filename. */
 #define DEFAULT_CONFIG_FILE "zebra.conf"
